src/main/archetype/pom.xml

```diff
@@ -512,8 +512,9 @@
             <!-- Apache Sling Dependencies -->
             <dependency>
                 <groupId>com.adobe.aem</groupId>
-                <artifactId>aem-api</artifactId>
-                <version>6.0.0.1</version>
+                <artifactId>uber-jar</artifactId>
+                <version>6.2.0</version>
+                <classifier>obfuscated-apis</classifier>
```

Comment:
Can you please change the classifier to just apis [0]. IMO as the uber-jar is now published on the public git repository repo.adobe.com there is now reason to only include the obfuscated one.

[0] https://docs.adobe.com/docs/en/aem/6-2/develop/dev-tools/ht-projects-maven.html#What%20is%20the%20UberJar?